src/xtgeo/cube/cube1.py

```diff
@@ -710,8 +710,7 @@ def from_file(self, sfile, fformat="guess", engine="segyio"):
             sfile (str): Filename (as string or pathlib.Path instance).
             fformat (str): file format guess/segy/rms_regular/xtgregcube
                 where 'guess' is default. Regard 'xtgrecube' format as experimental.
-            engine (str): For the SEGY reader, 'xtgeo' is builtin
-                while 'segyio' uses the SEGYIO library (default).
+            engine (str): Deprecated option, only engine='segyio' is supported.
```

Comment:
Add deprecation warning if someone specifies `engine`?

Reply:
I just missed this:
```
        if engine is not None:
            logger.warning("'engine' is a depracated option.")
        else:
            engine = "segyio"
```
so my bad

Reply:
No worries, I just added that code :D